deploy/charts/cert-manager/templates/webhook-validating-webhook.yaml

```diff
@@ -39,6 +39,7 @@ webhooks:
           - clusterissuers
           - certificaterequests
     failurePolicy: Fail
+    sideEffects: None
```

Comment:
This is housekeeping - our webhook has no side effects, so let the apiserver know this 😄